examples/retinanet/coco_eval.py

```diff
@@ -0,0 +1,339 @@
+from pycocotools.coco import COCO
```

Comment:
+1 for following the style guide. One problem is that we don't have a linter for this on GitHub.
Can we submit the CL as is and I fix the import order next week?

Reply:
I'll try to do this manually, but it might be the case that I miss something, or the order I use within these blocks is not correct. Without a linter, however, this is the best I can do.